class.jetpack.php

```diff
@@ -3212,6 +3212,166 @@ function admin_connect_notice() {
 
 		$dismiss_and_deactivate_url = wp_nonce_url( Jetpack::admin_url( '?page=jetpack&jetpack-notice=dismiss' ), 'jetpack-deactivate' );
 		?>
+
+		<div id="message" class="updated jp-wpcom-connect__container">
+			<div class="jp-wpcom-connect__inner-container">
```

Comment:
Are we closing this `div` properly? I'm not sure since line `3221` starts at the same indentation level.